test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -392,11 +392,7 @@ private Settings getRandomNodeSettings(long seed) {
             builder.put(SearchService.DEFAULT_KEEPALIVE_SETTING.getKey(), TimeValue.timeValueSeconds(100 + random.nextInt(5 * 60)));
         }
 
-        if (random.nextInt(10) == 0) {
-            // node gets an extra cpu this time
-            builder.put(EsExecutors.PROCESSORS, 1 + EsExecutors.boundedNumberOfProcessors(Settings.EMPTY));
-        }
-
+        builder.put(EsExecutors.PROCESSORS_SETTING.getKey(), 1 + random.nextInt(3));
```

Comment:
is the effect the same as before? we were changing this only rarely before while now we do it all the time?